### ISSUE TITLE
Update npm package as lib is outdated

### ISSUE THREAD

**Issue by pyros2097** [redux-loop/redux-loop#119]
I have installed redux-loop 2.2.2 and when I try to import createLoopStore it returns to me an undefined function.
Currently this is what my lib/index.js file shows in the redux-loop node module.,
```js
'use strict';

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.getEffect = exports.getModel = exports.liftState = exports.isLoop = exports.loop = exports.install = exports.Effects = exports.combineReducers = undefined;

var _loop = require('./loop');

var _effects = require('./effects');

var _install = require('./install');

var _combineReducers = require('./combineReducers');

var Effects = {
  constant: _effects.constant,
  promise: _effects.promise,
  call: _effects.call,
  batch: _effects.batch,
  none: _effects.none,
  lift: _effects.lift
};

exports.combineReducers = _combineReducers.combineReducers;
exports.Effects = Effects;
exports.install = _install.install;
exports.loop = _loop.loop;
exports.isLoop = _loop.isLoop;
exports.liftState = _loop.liftState;
exports.getModel = _loop.getModel;
exports.getEffect = _loop.getEffect;
```

**Comment by pyros2097** [comment]
Also in the first Quick Example you import createStore instead of CreateLoopStore
```js
import { createStore, Effects } from 'redux-loop';
```

**Comment by pyros2097** [comment]
Also wouldn't it be best if you exported a store enhancer instead or overriding createStore like redux-offline does it.

**Comment by ashishsc** [comment]
You should use the 2.2.2 branch for now. I'm pretty sure the master branch is currently being used at the dev branch, which is what confused me in https://github.com/redux-loop/redux-loop/issues/118 and https://github.com/redux-loop/redux-loop/issues/117

I linked to some discussions in pull requests there. I really don't think the master branch should have the 'potential' release merged in / have its readme, especially when there's no readme on npm.

**Comment by gpsamson** [comment]
@pyros2097 - did you ever get solve this issue?

**Comment by pyros2097** [comment]
@gpsamson Nope. I'm using freactal/baobab as my state container instead of redux.

**Comment by bdwain** [comment]
Hi everyone.

Sorry it's been so long since you've heard back. I'm going to be taking over maintenance of redux-loop. I'd like to get an update out in the next few days.

I know there have been a few proposed API changes in the last year. I've been using a fork of one those proposed version 3's for a while in projects that I've worked on (https://github.com/bdwain/redux-loop). I'd like to make that the starting point for any future changes. Luke and I discussed these changes last year and I think it'll be an improvement over version 2 without being too drastic of a change for people who are currently using it. It doesn't have typescript typings yet, but I would like to add that in the next few days as well.

The main idea behind the changes in my branch can be seen here [in this issue](https://github.com/redux-loop/redux-loop/issues/66) . It's passing action creators to effects/cmds instead of putting them in your side effect functions. Though since that discussion, i've consolidated some of the Cmd types.

I'll keep this issue open for now and close it when I've got everything updated. Sorry again for the delay!
